ui/js/react/treeherder/revision-list.jsx

```diff
@@ -0,0 +1,95 @@
+'use strict';
```

Comment:
Ed has a good point.  When we had a tiny amount of react, it perhaps made sense to section it off.  But now that we're going to be much more hybrid, putting them in their functional areas probably makes more sense.  Though doesn't need to happen in this PR, IMO.  :)